packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -0,0 +1,415 @@
+//TODO all file upload process needed to be checked and refactored with TS. skipping checks for passing CFD build
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import React from 'react';
+import { Formik, FormikErrors, FormikHelpers, FormikValues } from 'formik';
+import { DocumentUploadResponse } from '@deriv/api-types';
+import {
+    Loading,
+    Button,
+    FormSubmitErrorMessage,
+    Text,
+    ThemedScrollbars,
+    FormSubmitButton,
+    Modal,
+} from '@deriv/components';
+import { isMobile, validAddress, validPostCode, validLetterSymbol, validLength, getLocation, WS } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import FormFooter from '../../../Components/form-footer';
+import FormBody from '../../../Components/form-body';
+import FormBodySection from '../../../Components/form-body-section';
+import FormSubHeader from '../../../Components/form-sub-header';
+import LoadErrorMessage from '../../../Components/load-error-message';
+import LeaveConfirm from '../../../Components/leave-confirm';
+import FileUploaderContainer from '../../../Components/file-uploader-container';
+import CommonMistakeExamples from '../../../Components/poa/common-mistakes/common-mistake-examples';
+import PersonalDetailsForm from '../../../Components/forms/personal-details-form.jsx';
+import { validate } from '../../../Helpers/utils';
+import { isServerError } from '../../../Types/common.type';
+
+const FilesDescription = () => (
+    <div className='files-description'>
+        <Text size={isMobile() ? 'xxs' : 'xs'} as='div' className='files-description__title' weight='bold'>
+            {localize(
+                'We accept only these types of documents as proof of your address. The document must be recent (issued within last 6 months) and include your name and address:'
+            )}
+        </Text>
+        <ul>
+            <li>
```

Comment:
@yauheni-deriv let's map these li's based on their content :)

Reply:
Actually, I thought about it, but left as it is, because there are only 3 options.
but agree with you - will refactor it. thank you